featureFlag/index.html

```diff
@@ -3,9 +3,10 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="/images/index.ico" type="image/x-icon" />
     <link rel="stylesheet" href="../style.css" />
     <link rel="stylesheet" href="style.css" />
-    <title>Feature Flag</title>
+    <title>Feature Flag | Dashboard RDS</title>
```

Comment:
Why does this title look different than patterns followed on other pages?

Reply:
Means the full name of RDS should have been like 
`<title>Feature Flag | Dashboard Real Dev Squad</title>` ?

Reply:
have changed